src/components/Icon/AnalyzeDataIcon/AnalyzeDataIcon.jsx

```diff
@@ -0,0 +1,48 @@
+import _ from 'lodash';
+import React from 'react';
+import Icon from '../Icon';
+import { lucidClassNames } from '../../../util/style-helpers';
+import { createClass } from '../../../util/component-types';
+
+const cx = lucidClassNames.bind('&-AnalyzeDataIcon');
+
+/**
+ *
+ * {"categories": ["visual design", "icons"], "extend": "Icon", "madeFrom": ["Icon"]}
+ *
+ * Typically used to show the user that further data analysis is available.
+ */
+const AnalyzeDataIcon = createClass({
+	displayName: 'AnalyzeDataIcon',
+	propTypes: {
+		...Icon.propTypes,
+	},
+
+	render() {
+		const {
+			className,
+			...passThroughs,
+		} = this.props;
+
+		return (
+			<Icon
+				{...passThroughs}
+				{..._.pick(passThroughs, _.keys(Icon.propTypes))}
```

Comment:
i think it would make sense to have this line if the previous line used `omitProps`, but since these don't have anything besides `Icon.propTypes`, i think just `...passThroughs` works